send_conn.go

```diff
@@ -28,6 +28,9 @@ type sconn struct {
 	packetInfoOOB []byte
 	// If GSO enabled, and we receive a GSO error for this remote address, GSO is disabled.
 	gotGSOError bool
+	// Used to catch the error sometimes returned by the first sendmsg call on Linux,
+	// see https://github.com/golang/go/issues/63322.
+	wroteFirstPacket bool
```

Comment:
Is it confirmed that this happens only on the first packet? I didn't check this when I found that issue.

Reply:
I tested it, and it never occurred after the first write. I realize this is not a very satisfying answer, but to find this out we'd have to dive very deep into the Go / Linux networking stack.